client/menu/menu.js

```diff
@@ -13,7 +13,17 @@
 // limitations under the License.
 
 /**
- * @fileoverview Module for storing question and user data
+ * @fileoverview Adds an importable configuration file for the TIE menu page.
  */
 
-window.tieData = angular.module('tieData', []);
+window.tieMenu = angular.module('tieMenu', ['tieData']);
+
+tieMenu.controller('MenuPageController',
+  ['$scope', 'QuestionDataService', function($scope, QuestionDataService) {
+    var questionSetId = 'strings';
+    QuestionDataService.initCurrentQuestionSet(questionSetId);
+    $scope.questionSet = QuestionDataService.getCurrentQuestionSet(
```

Comment:
Yep, removing console.logs is absolutely the right thing to do here :)

When you extract into a directive, please pull this initialization logic into a function and write Karma tests for it.